src/metabase/util.cljc

```diff
@@ -532,22 +532,6 @@
        :cljs (js/Math.floor (/ (js/Math.log (abs x))
                                (js/Math.log 10))))))
 
-(defn update-if-exists
```

Comment:
The medley version of these functions are much more efficient, and we already use them more prevalently.